Automattic/wp-calypso
# Add sunrise color scheme

One part of #47036

#### Changes proposed in this Pull Request

* Add sunrise color scheme

#### Testing instructions

* Visit /me/account
* Choose sunrise and save

To see the wp-admin color scheme, create an atomic site, then visit /wp-admin/profile.php, choose color scheme and save.

Calypso:
![2020-11-05_16-02](https://user-images.githubusercontent.com/937354/98301448-5e1a7800-1f80-11eb-8b77-8903fd072b57.png)

Wp-Admin:
![2020-11-05_16-02_1](https://user-images.githubusercontent.com/937354/98301458-62469580-1f80-11eb-9f03-dfda175ff93c.png)

#### Notes

If you're following along with Ectoplasm from #47122, you'll see I generated a palette of "Ectoplasm green" in -10 to -90 values.  For this one, I found that "Studio Orange" was a close enough match for primary+accent colors and reused it.  The hue of the highlight color in the wp-admin scheme is 25, and studio orange is a gradient between 25 and 35.  The sidebar and button colors should still match exactly, though.
